src/20231122_AaveV3Ethereum_GHO_Incident_Report/AaveV3Ethereum_GHO_Incident_Report_20231122.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import 'forge-std/console.sol';
```

Comment:
we can remove all the unused imports here

Reply:
removed in e56cb307e99a440720a041a83f400bc0f14caf7a